### ISSUE TITLE
Review not approved by the assignee when it is approved

### ISSUE THREAD

**Issue by ignatenkobrain** [fedora-infra/pkgdb2#308]
![screenshot from 2015-12-28 14-32-33](https://cloud.githubusercontent.com/assets/2866862/12019387/eb670314-ad6f-11e5-9d2f-c0701bc6cc28.png)
![screenshot from 2015-12-28 14-32-41](https://cloud.githubusercontent.com/assets/2866862/12019388/eb93b2ec-ad6f-11e5-9e16-b80a4e401347.png)

**Comment by ignatenkobrain** [comment]
assignee is the same Lubomir Rintel, but with @redhat.com

**Comment by pypingou** [comment]
I guess Lubomir has an exception allowing him to use a different email in FAS and in bugzilla, that's indeed something that pkgdb2 doesn't know atm.

**Comment by ralphbean** [comment]
So pkgdb would need to be updated to know about those exceptions, correct?  We keep that mapping somewhere... is it in python-fedora?

**Comment by pypingou** [comment]
Yes to both and it's the pkgdb hosts that has the patch version of python-fedora for these, so that should make things easier

**Comment by yash14123** [comment]
Hey! Please assign me this issue.

**Comment by pypingou** [comment]
@yash14123 you have indicated on #311 that you are working on it, so which is it?
Let's take the issues one at a time to leave some for others. That is not to say that if you fixed this one already (ie: when there is a PR fixing it) you can't pick up another one. But let's just not take them all preventing other people from working on them.
